Basic/Primitive/primitive.pd

```diff
@@ -2179,9 +2179,45 @@ As of 2.062, the generator changed from Perl's generator to xoshiro256++
 
 =for usage
 
+From within pdl:
```

Comment:
Please change this to say "within `C<perldl>`", because "within pdl" is very unclear :-)